### ISSUE TITLE
Click targets too small

### ISSUE THREAD

**Issue by Simon-Laux** [pixelfed/pixelfed-rn#221]
Some buttons have a very small click area, so they are hard to tap. Others are small and unintuitive.

You can inspect some with the element inspector (run `npm run start`, press `m` then open element inspector, then enable `Touchables`.

# What I found so far:

- [x] Post -> comments button (number part should also be part of click target, and click target should be larger, maybe also include padding/margin)
- [x] view all comments click area could also be bigger / include some of the space around
- [x] Buttons in top bar should also have bigger click area
- [x] counter on like and reblog buttons are extra buttons, their click target is also way to small, it's hard to hit them

<img src=https://github.com/user-attachments/assets/2e4299e5-0cba-458c-a760-0ba070a9f3dd width=400 />

## Comment Feed

   - [x] Like button - too small width, but to high height
   - [x] Profile button - too high, but width is fine
   - [x] Commenting: privacy and CW options -> especially when keyboard is shown these "buttons" are hard to hit

<img src=https://github.com/user-attachments/assets/48e7aa0d-d5be-46e2-859f-f1efc00d5fe3 width=400 />

## Own Profile Page

- "Tabs"
  - [x] click area should be as big as possible, like the tabs at bottom of the screen.
  - off-topic: ~~I don't know what the bookmark tab is for~~ nvm, the option is just missing in mobile app: https://github.com/pixelfed/pixelfed-rn/issues/222

<img src=https://github.com/user-attachments/assets/6193880e-df4d-4d7a-a899-755b3e3d70cd width=400 />

## Profile page of other accounts

- [x] Back button

## Notifications page

Edit: moved to new issue, because there are still some open questions on how the improvement would look like:
-  #402
